debian/rules

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/make -f
 
 %:
-	dh $@ --with=systemd
+	dh $@
```

Comment:
Won't this change remove the `systemd` related scripts? The `--with=systemd` line add some useful install/uninstall scripts

Reply:
In compat level 10 (I think), this is automatically included. It actually complains if you still have it. You can see that this produces working builds (using the CI PR that I provided): https://github.com/pi-top/touchegg/runs/1689662938?check_suite_focus=true#step:6:1586